src/db/addTermMgmt.sql

```diff
@@ -195,10 +196,10 @@ CREATE OR REPLACE FUNCTION getTermSectionCount(termID INT)
 RETURNS INT
 AS
 $$
-BEGIN
-   RAISE WARNING 'Function not implemented';
-END
-$$ LANGUAGE plpgsql
+    SELECT COUNT(DISTINCT Section)
```

Comment:
```suggestion
    SELECT COUNT(Section)
```

I recommend removing the `DISTINCT` keyword, since sections will already be distinct in this table.